src/menu.c

```diff
@@ -39,13 +38,17 @@ int _display_menu(char **options, int options_len, char *menu_text, char *help_t
                 return -1;
             }
             if (strstr(input, "!h") != NULL) {
-                printf("%s\n", help_text);
+                printf("%s\n\n", help_text);
```

Comment:
!q and !h currently triggers if the input contains those letters anywhere in the string. It can be fixed by comparing the strstr return pointer to the input pointer, but then would fail for an input like "!help me i'm screaming with '!' at both ends because i'm spanish but can't type ! upside down!".

Just add a check like strlen(input) == 2